src/views/virtualmachines/details/tabs/configuration/network/components/list/NetworkInterfaceRow.tsx

```diff
@@ -21,31 +23,55 @@ const NetworkInterfaceRow: FC<
   >
 > = ({ activeColumnIDs, obj: { iface, network }, rowData: { isPending, vm } }) => {
   const { t } = useKubevirtTranslation();
+  const nicDisabled = isStopped(vm) && iface?.state === 'absent';
 
   return (
     <>
-      <TableData activeColumnIDs={activeColumnIDs} id="name">
+      <TableData
+        activeColumnIDs={activeColumnIDs}
+        className={classNames({ 'pf-u-disabled-color-100': nicDisabled })}
```

Comment:
you can make this
const className = classNames({ 'pf-u-disabled-color-100': nicDisabled }) at top level
and pass it to everyone className={className}
for the last one, actions u can join ['dropdown-kebab-pf pf-v5-c-table__action', className ].join(' ')
if u want to make the change awesome, if not ill lgtm it as it is

Reply:
Nice! I've missed this when reviewing.. IMO creating a new const and just passing it where needed is definitely much better here.

Reply:
I agree, Hilda! It's much cleaner in the end. Thanks, Matan!